cross/ndpi/PLIST

```diff
@@ -0,0 +1,6 @@
+bin:bin/ndpiReader
+lnk:lib/libndpi.so
+lnk:lib/libndpi.so.3
+lib:lib/libndpi.so.3.0.0
+rsc:sbin/ndpi/ndpiCustomCategory.txt
+rsc:sbin/ndpi/ndpiProtos.txt
```

Comment:
Strange location for these text files. Why `sbin`? which is often meaning for "system binaries"... These files look like resource (so in `share/`) or a configuration files (so in `etc/`)

Reply:
for an unknown reason this example files are installed to sbin/ndpi
will remove these